recipes-devtools/partition-utils/qcom-ptool/0001-ptool.py-Generate-zero-files-in-output-folder-when-s.patch

```diff
@@ -8,6 +8,8 @@ The patch allows ptool to generate zerosector files in output directory
 when such directory path is specified as argument to -t option.
 
 Upstream-Status: Pending
+
```

Comment:
There should be no empty lines between tags IIRC

Reply:
I think there are no rule for this in or-core, with `grep` we can see that all combinations with and without spaces before and after are used.